<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-181TTAT" failures="4" tests="36" name="Test_All_Modules_chrome" time="219.345" errors="0" timestamp="2025-08-17T00:37:46 GMT-04:00">
  <testcase classname="testCases.StackTest" name="open_stac_page" time="0.494"/>
  <testcase classname="testCases.QueueTest" name="open_queue_page" time="0.483"/>
  <testcase classname="testCases.StackTest" name="stackOperations" time="1.238"/>
  <testcase classname="testCases.QueueTest" name="queueImplementation" time="0.847"/>
  <testcase classname="testCases.StackTest" name="stackoperationsCodeEditor" time="1.829"/>
  <testcase classname="testCases.QueueTest" name="queueImplementationCodeEditor" time="1.562"/>
  <testcase classname="testCases.StackTest" name="StackTryingEmptyEditor" time="2.22">
    <skipped/>
  </testcase> <!-- StackTryingEmptyEditor -->
  <testcase classname="testCases.StackTest" name="StackTryingEmptyEditor" time="2.352">
    <skipped/>
  </testcase> <!-- StackTryingEmptyEditor -->
  <testcase classname="testCases.StackTest" name="StackTryingEmptyEditor" time="1.921">
    <skipped/>
  </testcase> <!-- StackTryingEmptyEditor -->
  <testcase classname="testCases.StackTest" name="StackTryingEmptyEditor" time="2.56">
    <failure type="java.lang.AssertionError" message="Failing this test case to show the bug which is, no alert message comes up when clicking on run button without entering anhy code in it">
      <![CDATA[java.lang.AssertionError: Failing this test case to show the bug which is, no alert message comes up when clicking on run button without entering anhy code in it
at testCases.StackTest.StackTryingEmptyEditor(StackTest.java:78)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- StackTryingEmptyEditor -->
  <testcase classname="testCases.QueueTest" name="queueTryingEmptyEditor" time="2.108">
    <skipped/>
  </testcase> <!-- queueTryingEmptyEditor -->
  <testcase classname="testCases.QueueTest" name="queueTryingEmptyEditor" time="1.904">
    <skipped/>
  </testcase> <!-- queueTryingEmptyEditor -->
  <testcase classname="testCases.QueueTest" name="queueTryingEmptyEditor" time="2.127">
    <skipped/>
  </testcase> <!-- queueTryingEmptyEditor -->
  <testcase classname="testCases.QueueTest" name="queueTryingEmptyEditor" time="2.424">
    <failure type="java.lang.AssertionError" message="Failing this test case to show the bug which is, no alert message comes up when clicking on run button without entering anhy code in it">
      <![CDATA[java.lang.AssertionError: Failing this test case to show the bug which is, no alert message comes up when clicking on run button without entering anhy code in it
at testCases.QueueTest.queueTryingEmptyEditor(QueueTest.java:80)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- queueTryingEmptyEditor -->
  <testcase classname="testCases.StackTest" name="StackTryingValidAndInvalidCode" time="2.25"/>
  <testcase classname="testCases.StackTest" name="StackTryingValidAndInvalidCode" time="2.052"/>
  <testcase classname="testCases.QueueTest" name="queueTryingValidAndInvalidCode" time="2.042"/>
  <testcase classname="testCases.QueueTest" name="queueTryingValidAndInvalidCode" time="1.725"/>
  <testcase classname="testCases.StackTest" name="stackimplementation" time="1.136"/>
  <testcase classname="testCases.QueueTest" name="queueImplementationUsingCollection" time="0.865"/>
  <testcase classname="testCases.StackTest" name="stackimplementationCodeEditor" time="1.627"/>
  <testcase classname="testCases.QueueTest" name="queueImplementationUsingCollectionCodeEditor" time="2.327"/>
  <testcase classname="testCases.StackTest" name="stackapplication" time="2.022"/>
  <testcase classname="testCases.QueueTest" name="queueImplementationUsingArray" time="1.81"/>
  <testcase classname="testCases.StackTest" name="stackapplicationCodeEditor" time="2.218"/>
  <testcase classname="testCases.QueueTest" name="queueImplementationUsingArrayCodeEditor" time="2.445"/>
  <testcase classname="testCases.StackTest" name="StackopenPracticeQuestions" time="1.919">
    <skipped/>
  </testcase> <!-- StackopenPracticeQuestions -->
  <testcase classname="testCases.StackTest" name="StackopenPracticeQuestions" time="2.569">
    <skipped/>
  </testcase> <!-- StackopenPracticeQuestions -->
  <testcase classname="testCases.StackTest" name="StackopenPracticeQuestions" time="2.0">
    <skipped/>
  </testcase> <!-- StackopenPracticeQuestions -->
  <testcase classname="testCases.StackTest" name="StackopenPracticeQuestions" time="2.384">
    <failure type="java.lang.AssertionError" message="Failing this test case to show the bug which is,there are no practice questions available for stack module">
      <![CDATA[java.lang.AssertionError: Failing this test case to show the bug which is,there are no practice questions available for stack module
at testCases.StackTest.StackopenPracticeQuestions(StackTest.java:160)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- StackopenPracticeQuestions -->
  <testcase classname="testCases.QueueTest" name="queueOperations" time="1.587"/>
  <testcase classname="testCases.QueueTest" name="queueOperationsCodeEditor" time="2.122"/>
  <testcase classname="testCases.QueueTest" name="queueOpenPracticeQuestions" time="2.184">
    <skipped/>
  </testcase> <!-- queueOpenPracticeQuestions -->
  <testcase classname="testCases.QueueTest" name="queueOpenPracticeQuestions" time="1.993">
    <skipped/>
  </testcase> <!-- queueOpenPracticeQuestions -->
  <testcase classname="testCases.QueueTest" name="queueOpenPracticeQuestions" time="2.996">
    <skipped/>
  </testcase> <!-- queueOpenPracticeQuestions -->
  <testcase classname="testCases.QueueTest" name="queueOpenPracticeQuestions" time="1.021">
    <failure type="java.lang.AssertionError" message="Failing this test case to show the bug which is,there are no practice questions available for queue module">
      <![CDATA[java.lang.AssertionError: Failing this test case to show the bug which is,there are no practice questions available for queue module
at testCases.QueueTest.queueOpenPracticeQuestions(QueueTest.java:181)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- queueOpenPracticeQuestions -->
</testsuite> <!-- Test_All_Modules_chrome -->
